### ISSUE TITLE
Option for tuning fork watches

### ISSUE THREAD

**Issue by jrfantom** [vacaboja/tg#39]
I have a few acutrons that need to be regulated and I think this would be the perfect tool

**Comment by lachesis** [comment]
Yeah that'd be pretty neat. I guess you could add a 1296000 bph entry.
